### ISSUE TITLE
Unexpected error

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/the-new-hotness#171]
hotness told me to do this:

https://bugzilla.redhat.com/show_bug.cgi?id=1435488

**Comment by proski** [comment]
I got it too: https://bugzilla.redhat.com/show_bug.cgi?id=1437053

I wonder why I was asked on Bugzilla to report the issue. Why cannot "the new hotness" report its own issues right here on Github without asking anyone for help?

**Comment by zdohnal** [comment]
I got this error too on this bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1437270

**Comment by jeremycline** [comment]
@bowlofeggs it looks like there was a list index out of range somehow, but that code has since been refactored and is gone now.

> I wonder why I was asked on Bugzilla to report the issue. Why cannot "the new hotness" report its own issues right here on Github without asking anyone for help?

I inherited this project and then added some additional error handling. At the time I didn't realize it was configured to email tracebacks that are logged. The error reporting gets a lot better in the (as yet not in production) 0.8.1 and will get even better when I can find a few minutes to improve it with all the various errors I've seen. Sorry for the noise.

For xorgxrdp, it was that release-monitoring.org was announcing the new versions with a v prefix and the-new-hotness puts that in the version field (which then gets dropped into the source URL) and results in a 404. I went ahead and fixed the project in release-monitoring.org (and there's work upstream to make this automatic for all projects).

For the cups update, it seems like it got the new version for the old version, which is odd. Typically it gets the old version twice when the version macro isn't in the URL.
```
spectool was unable to grab new sources

old source: cups-2.2.3-source.tar.gz
old sha256: 66701fe15838f2c892052c913bde1ba106bbee2e0a953c955a62ecacce76885f

new source: ./cups-2.2.3-source.tar.gz
new sha256: 66701fe15838f2c892052c913bde1ba106bbee2e0a953c955a62ecacce76885f
```
Based on the commit history of cups, it looks like you were faster than the normal release-monitoring.org scan and had already updated the specfile to 2.2.3. We should probably handle that possibility.
